extensions/wikia/WallNotifications/GlobalNavigationWallNotificationsController.class.php

```diff
@@ -6,11 +6,23 @@
 
 
 class GlobalNavigationWallNotificationsController extends WallNotificationControllerBase {
-	protected function addAssets() {
-		$this->response->addAsset( 'extensions/wikia/GlobalNavigation/styles/GlobalNavigationNotifications.scss' );
-		$this->response->addAsset( 'wall_notifications_global_navigation_js' );
+	const NOTIFICATION_TITLE_LIMIT = 48;
+
+	private $wallHelper;
+
+	public function __construct() {
+		parent::__construct();
+		$this->app = F::App();
```

Comment:
Controllers already has `app` defined. 
BTW I see a lot of changes in this file that does nothing ;)